src/screens/InfoTodo.tsx

```diff
@@ -0,0 +1,54 @@
+import { StyleSheet, Text, View } from 'react-native'
+
+import { ClearButton } from '../components/ClearButton'
+import { InfoTodoNavProps } from '../navigation/types'
+import { Colors } from '../styles/colors'
+
+export default function InfoTodo({ navigation, route }: InfoTodoNavProps) {
+  const { id, title, description, checked } = route.params
+
+  function returnHomeScreen() {
+    navigation.navigate('HomeScreen', {
+      doneItem: {
+        id: id,
```

Comment:
When the variable name and the key have the same name you can do this: 
```suggestion
        id,
```